recipes/greenlearning/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "greenlearning" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/NBoulle/greenlearning/archive/v{{ version }}.tar.gz"
+  sha256: 7d7d53d5d02d2123add2a58a04efebb6103bab1f2203fda2b287507fa50c5caa
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - matplotlib
```

Normally `-base` is sufficient. `matplotlib` is only needed if the Qt backend is a hard requirement.
```suggestion
    - matplotlib-base
```